src/tests/sqlsmith/src/runner.rs

```diff
@@ -304,8 +306,7 @@ async fn test_batch_queries<R: Rng>(
         test_session_variable(client, rng).await;
         let sql = sql_gen(rng, tables.clone());
         tracing::info!("[EXECUTING TEST_BATCH]: {}", sql);
```

Comment:
These logs should be included in `run_query` too to clean up the code. Do so in other PRs.